src/server/pfs/fuse/filesystem.go

```diff
@@ -333,11 +333,13 @@ func (f *file) delimiter() pfsclient.Delimiter {
 }
 
 func (f *file) touch() error {
```

Comment:
Does this mean that if you save a file as `foo.json` in a FUSE mount, we somehow end up with a directory `foo`?